examples/metalk8s-solution-example/operator/pkg/config/config.go

```diff
@@ -0,0 +1,39 @@
+package config
+
+import (
+	"fmt"
+	"io/ioutil"
+	"gopkg.in/yaml.v2"
+)
+
+type TOperatorConfig struct {
+        Kind         string `yaml:"kind"`
+        APIVersion   string `yaml:"apiVersion"`
```

Comment:
Those values should be checked before the complete `yaml.Unmarshal`, to ensure we understand the provided format and warn the user properly if we don't.

Reply:
I've added a validator, tell me if you have a better idea in mind.